install_script_op_worker.sh

```diff
@@ -0,0 +1,572 @@
+    OS="RedHat"
+# Some newer distros like Amazon may not have a redhat-release file
+elif [ -f /etc/system-release ] || [ "$DISTRIBUTION" == "Amazon" ]; then
+    OS="RedHat"
+fi
+
+# Root user detection
+if [ "$(echo "$UID")" = "0" ]; then
+    sudo_cmd=''
+else
+    sudo_cmd='sudo'
+fi
+
+# Install the necessary package sources
+if [ "$OS" = "RedHat" ]; then
+    echo -e "\033[34m\n* Installing YUM sources for Datadog\n\033[0m"
+
+    UNAME_M=$(uname -m)
+    if [ "$UNAME_M" == "aarch64" ]; then
+        ARCHI="aarch64"
+    else
+        ARCHI="x86_64"
+    fi
+
+    # Because of https://bugzilla.redhat.com/show_bug.cgi?id=1792506, we disable
+    # repo_gpgcheck on RHEL/CentOS 8.1
+    if [ -z "$rpm_repo_gpgcheck" ]; then
+        if grep -q "8\.1\(\b\|\.\)" /etc/redhat-release 2>/dev/null; then
+            rpm_repo_gpgcheck=0
+        else
+            rpm_repo_gpgcheck=1
+        fi
+    fi
+
+    gpgkeys="https://${keys_url}/DATADOG_RPM_KEY.public\n       https://${keys_url}/DATADOG_RPM_KEY_E09422B3.public"
+
+    gpgkeys=''
+    separator='\n       '
+    for key_path in "${RPM_GPG_KEYS[@]}"; do
+      gpgkeys="${gpgkeys:+"${gpgkeys}${separator}"}https://${keys_url}/${key_path}"
+    done
+
+    $sudo_cmd sh -c "echo -e '[observability-pipelines-worker]\nname = Observability Pipelines Worker\nbaseurl = https://${yum_url}/${yum_version_path}/${ARCHI}/\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=${rpm_repo_gpgcheck}\npriority=1\ngpgkey=${gpgkeys}' > /etc/yum.repos.d/datadog-observability-pipelines-worker.repo"
+
+    $sudo_cmd yum -y clean metadata
+
+    dnf_flag=""
+    if [ -f "/usr/bin/dnf" ] && { [ ! -f "/usr/bin/yum" ] || [ -L "/usr/bin/yum" ]; } ; then
+      # On modern Red Hat based distros, yum is an alias (symlink) of dnf.
+      # "dnf install" doesn't upgrade a package if a newer version
+      # is available, unless the --best flag is set
+      # NOTE: we assume that sometime in the future "/usr/bin/yum" will
+      # be removed altogether, so we test for that as well.
+      dnf_flag="--best"
+    fi
+
+    if [ -n "$worker_minor_version" ]; then
+        # Example: observability-pipelines-worker-1.2.1-1
+        pkg_pattern="$worker_major_version\.${worker_minor_version%.}(\.[[:digit:]]+){0,1}(-[[:digit:]])?"
+        agent_version_custom="$(yum -y --disablerepo=* --enablerepo=observability-pipelines-worker list --showduplicates observability-pipelines-worker | sort -r | grep -E "$pkg_pattern" -om1)" || true
+        verify_worker_version "-"
+    fi
+
+    declare -a packages
+    packages=("$worker_flavor")
+    
+    echo -e "  \033[33mInstalling package(s): ${packages[*]}\n\033[0m"
+
+    $sudo_cmd yum -y --disablerepo='*' --enablerepo='observability-pipelines-worker' install $dnf_flag "${packages[@]}" || $sudo_cmd yum -y install $dnf_flag "${packages[@]}"
+
+elif [ "$OS" = "Debian" ]; then
```

Comment:
```suggestion
elif [ "$OS" == "Debian" ]; then
```

Reply:
Addressed in https://github.com/DataDog/agent-linux-install-script/pull/66/commits/5b76b4d796bcc42eb3364ed7cd8fc9e9a33fe997/https://github.com/DataDog/agent-linux-install-script/pull/66/commits/707be0c4f2c8d66fe3ad78290f755dd6102a5725